projects/canopy/src/lib/orientation/docs/orientation.stories.ts

```diff
@@ -0,0 +1,81 @@
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+
+import { LgCardModule } from '../../card';
+import { LgOrientationModule } from '../orientation.module';
+import { LgOrientationDirective } from '../orientation.directive';
+import { LgPaddingModule } from '../../spacing/padding/padding.module';
+
+const responsiveCategory = 'Responsive';
+
+export default {
+  title: 'Helpers/Directives/Orientation/Examples',
+  decorators: [
+    moduleMetadata({
+      imports: [ LgOrientationModule, LgCardModule, LgPaddingModule ],
+    }),
+  ],
+  parameters: {
+    a11y: {
```

Comment:
You will also need to add: 
```
    viewMode: 'story',
    previewTabs: { 'storybook/docs/panel': { hidden: true } },
```
otherwise the `docs` tab will appear too (which we don't want).

Reply:
Done. :)